examples/src/components/DateFilterWithColumnChartExample.jsx

```diff
@@ -0,0 +1,157 @@
+            type: "absolutePreset",
+        },
+        {
+            from: "2016-01-01",
+            to: "2016-12-31",
+            name: "Year 2016",
+            localIdentifier: "year2016",
+            visible: true,
+            type: "absolutePreset",
+        },
+        {
+            from: "2017-01-01",
+            to: "2017-12-31",
+            name: "Year 2017",
+            localIdentifier: "year2017",
+            visible: true,
+            type: "absolutePreset",
+        },
+    ],
+    relativeForm: {
+        localIdentifier: "RELATIVE_FORM",
+        type: "relativeForm",
+        granularity: "GDC.time.year",
+        from: -2,
+        to: -2,
+        name: "",
+        visible: true,
+        availableGranularities,
+    },
+    relativePreset: {
+        "GDC.time.year": [
+            {
+                from: -2,
+                to: -2,
+                granularity: "GDC.time.year",
+                localIdentifier: "twoYearsAgo",
+                type: "relativePreset",
+                visible: true,
+                name: "2 years ago",
+            },
+            {
+                from: -3,
+                to: -3,
+                granularity: "GDC.time.year",
+                localIdentifier: "threeYearsAgo",
+                type: "relativePreset",
+                visible: true,
+                name: "3 years ago",
+            },
+            {
+                from: -4,
+                to: -4,
+                granularity: "GDC.time.year",
+                localIdentifier: "fourYearsAgo",
+                type: "relativePreset",
+                visible: true,
+                name: "4 years ago",
+            },
+        ],
+    },
+};
+
+export class DateFilterComponentExample extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedFilterOption: defaultDateFilterOptions.allTime,
+            excludeCurrentPeriod: false,
+        };
+    }
+
+    onApply = (dateFilterOption, excludeCurrentPeriod) => {
+        this.setState({
+            selectedFilterOption: dateFilterOption,
+            excludeCurrentPeriod,
+        });
+    };
+
+    render() {
+        const dateFilter = DateFilterHelpers.mapOptionToAfm(this.state.selectedFilterOption, {
+            identifier: dateDatasetIdentifier,
+        });
+
+        return (
+            <div>
+                <div style={{ width: 200 }}>
+                    <DateFilter
+                        excludeCurrentPeriod={this.state.excludeCurrentPeriod}
+                        selectedFilterOption={this.state.selectedFilterOption}
+                        filterOptions={defaultDateFilterOptions}
+                        availableGranularities={availableGranularities}
+                        customFilterName="Selected date range"
+                        dateFilterMode="active"
+                        onApply={this.onApply}
+                        onCancel={this.onCancel}
```

Comment:
Seems to me like `this.onCancel`, `this.onOpen` and `this.onClose` are undefined here.